### PR TITLE
mw.msg can take multiple parameters

### PR DESCRIPTION
**Pull request by jdlrobson**
e.g mw.msg( 'key', 'Argument 1, 'Argument 2')

### CODE REVIEW

**Comment by jdlrobson** [comment]
@DannyS712 - can you sanity check this pull request and the other three for me? Thanks in advance.

File: MediaWiki.d.ts [hunk 1]
```diff
@@ -141,7 +141,7 @@ interface MediaWiki {
 	 *
 	 * @param messageName i18n message name
 	 */
-	msg( messageName: string|null ): string;
+	msg( messageName: string|null, ...args: string[] ): string;
```
Comment (DannyS712) [inline MediaWiki.d.ts:144]: should have @param documentation, and its not just strings (core documents it as `{...Mixed}`)

**Review by DannyS712 — CHANGES_REQUESTED** [review]

File: MediaWiki.d.ts (hunk 1 above)
Comment (jdlrobson) [reply MediaWiki.d.ts]: Done.

Comment (DannyS712) [reply MediaWiki.d.ts]: If I go to https://github.com/wikimedia/typescript-types/pull/11/files its still missing the @param - am I missing something?

Comment (jdlrobson) [reply MediaWiki.d.ts]: Nope I just forgot to commit :)

**Pull request merged**